system-tests/dashboard/test-dashboard.js

```diff
@@ -0,0 +1,205 @@
+require("../_support/utils/ServicesUtil");
+const { Timeouts } = require("../_support/constants");
+
+// same parameters as defined in app.json
+const SERVICE_NAME = "dashboard-test-service";
+const SERVICE_CPUS = 0.1;
+const SERVICE_MEM = 10;
+const SERVICE_DISK = 10;
+
+function deleteService() {
```

Comment:
Shall we extract these helper functions into `system-tests/_support`? I think we might want to reuse them.